cli.js

```diff
@@ -39,12 +41,23 @@ function render() {
 		return;
 	}
 
-	logUpdate([
+	var output = [
 		'',
 		'      Ping ' + getSpinner('ping') + ' ' + stats.ping,
 		'  Download ' + getSpinner('download') + ' ' + stats.download,
 		'    Upload ' + getSpinner('upload') + ' ' + stats.upload
-	].join('\n'));
+	];
+
+	if (cli.flags.verbose && stats.data) {
+		output = output.concat([
+			'',
+			'    Server   ' + chalk.cyan(stats.data.server.host),
+			'  Location   ' + chalk.cyan(stats.data.server.location + chalk.dim(' (' + stats.data.server.country + ')')),
+			'  Distance   ' + chalk.cyan(roundTo(stats.data.server.distance, 1) + chalk.dim(' km'))
+		]);
```

Comment:
When the verbose data is not available yet, I think we should output newlines the amount of the lines, so it doesn't jump when the information becomes available.